Web/View/LandingPages/Edit.hs

```diff
@@ -66,11 +66,7 @@ orderAndRenderParagraphs ctas quotes =
             [hsx|
                 <li class="flex flex-row gap-2 items-baseline">
 
-                    <div class="sortable-handle">
-                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-4 h-4 text-gray-200 hover:text-gray-500">
-                            <path stroke-linecap="round" stroke-linejoin="round" d="M3 7.5L7.5 3m0 0L12 7.5M7.5 3v13.5m13.5 0L16.5 21m0 0L12 16.5m4.5 4.5V7.5" />
-                        </svg>
-                    </div>
+                    {sortableHandle}
```

Comment:
When adding this we get an error

![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/4cf93186-9020-4301-b073-2f79a5632f65)

Reply:
If I replace below `length allSorted > 1` with `length [1..10] > 1` it compiles. What's weird is that if I keep `length allSorted > 1` but not have it in the `hsx` there's no error.